lib/ts/recipe/session/utils.ts

```diff
@@ -211,10 +211,13 @@ export function validateAndNormaliseUserInput(
 
     let enableJWT = false;
     let accessTokenPayloadJWTPropertyName = "jwt";
+    // By default the issuer should be apiDomain + apiBasePath
+    let jwtIssuer = appInfo.apiDomain.getAsStringDangerous() + appInfo.apiBasePath.getAsStringDangerous();
```

Comment:
This step should actually be done in the open id normalisation. Not here.

Reply:
Done